pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -1789,11 +1798,25 @@ public void RestoreCoverViewOnScreen(bool hasSavedView = true)
             if (hasSavedView)
                 RestoreViewNameOnScreen("cover");
             // Make sure Skyline is the standard size for a cover shot - Window size and screen shot size differ
-            // And center it in the screen to have the best chance of not needing to move it before Alt-PtrSc
+            SetSkylineWindowSize(1200, 800);
+        }
+
+        // Make the Skyline window as large as possible, without actually putting it into
+        // Maximized state which prevents further resizing
+        const int marginW = 14;
+        const int marginH = 7;
+        public void MaximizeSkylineWindow()
+        {
+            SetSkylineWindowSize(int.MaxValue - marginW, int.MaxValue - marginH);
+        }
+
+        // Set the Skyline window size, and center it in the screen to have the best chance of not needing to move it before Alt-PtrSc
+        public void SetSkylineWindowSize(int width, int height)
+        {
             RunUI(() =>
             {
-                var skylineSize = new Size(1200 + 14, 800 + 7);
                 var screenRect = Screen.FromControl(SkylineWindow).Bounds;
+                var skylineSize = new Size(Math.Min(screenRect.Width, width + marginW), Math.Min(screenRect.Height,  height + marginH)); // So we don't roll into negative territory when SetSkylineWindowSize adds margins
```

Comment:
Probably would prefer to fail the test, if these go negative. Or better yet, fail if the screenRect is not 1920x1080 and 100% scaling. This is an assumption of taking cover shots that I did not put into code, because I was the only person taking them and I knew they needed to be done on a 1920x1080 screen at 100% scaling.

Reply:
Easy enough to fail if screen is too small. Getting screen scaling turns out to be a mess, I don't think it's worth the trouble, especially given that other parts of our dev process already demand this be at 100%.

Reply:
I'll put the CoverShot related changes on their own branch then merge that.

Reply:
Yes. I think just fail early in covershot mode, if the screen is not 1920x1080. No guarantees you would get a valid covershot without that exact screen resolution. Probably unlikely anyone would use greater than 100% at that resolution either.

Reply:
>>  fail early in covershot mode, if the screen is not 1920x1080

OK, will do